### PR TITLE
[IMP] add defaut group by account / tax code + add form view + add invoi...

### PR DESCRIPTION
**Pull request by vrenaville**
This Improve :
- add defaut group by account / tax code
- add posibility to switch to form view 
- add invoice link in tree view

### CODE REVIEW

File: account_tax_analysis/account_tax_analysis_view.xml [hunk 1]
```diff
@@ -25,21 +25,31 @@
...
+            	    name="group_by_period"
                     icon="terp-go-month"
                     domain="[]"
                     context="{'group_by': 'period_id'}"/>
             <filter string="Journal"
+                    name="group_by_journal"
                     domain="[]"
                     context="{'group_by': 'journal_id'}"/>
+			<filter string="Account / Tax"
+            	    name="group_by_account_tax"
```
Comment (pedrobaeza) [inline account_tax_analysis/account_tax_analysis_view.xml]: strange indentation

File: account_tax_analysis/account_tax_analysis_view.xml [hunk 2]
```diff
@@ -25,21 +25,31 @@
...
+				    icon="terp-folder-green"
+				    domain="[]"
+				    context="{'group_by': ['tax_code_id', 'account_id'] }"/>
             <filter string="Tax account"
+            	    name="group_by_tax"
```
Comment (pedrobaeza) [inline account_tax_analysis/account_tax_analysis_view.xml]: strange indentation

**Comment by pedrobaeza** [comment]
Add translations

File: account_tax_analysis/account_tax_analysis_view.xml (hunk 1 above)
Comment (sbidoul) [inline account_tax_analysis/account_tax_analysis_view.xml]: The filter is more Tax / Account than Account / Tax. But it's really a detail.

**Comment by sbidoul** [comment]
Except Pedro's comment, :+1: 

Nice improvement.

File: account_tax_analysis/account_tax_analysis_view.xml (hunk 1 above)
Comment (yvaucher) [inline account_tax_analysis/account_tax_analysis_view.xml]: There are tabs in that indentation

**Comment by fclementic2c** [comment]
:+1:

**Comment by adrienpeiffer** [comment]
:+1:

**Pull request merged**